### ISSUE TITLE
YAML framework should distinguish between a missing `ot:` and `ot: null`

### ISSUE THREAD

**Issue by VeXocide** [rethinkdb/rethinkdb#6102]
The YAML framework should distinguish between a missing `ot:` and `ot: null`.

```
    - cd: r.db("rethinkdb").table("permissions").get([])
      ot: null
```

This should check that the output is equivalent to`None` in Python, it currently gets translated to `anything()`

**Comment by larkost** [comment]
This is up for review in CR 3755 from branch `larkost/6102-allow-null-in-yaml-output`.

**Comment by larkost** [comment]
This has been merged into `next` with cf334060. This includes fixed for a few tests that incorrectly used the handling of `null` (or falsy values).
